ndc-client/tests/json_schema/schema_response.jsonschema

```diff
@@ -0,0 +1,465 @@
+      "description": "A list of object types which can be used as the types of arguments, or return types of procedures. Names should not overlap with collection names or scalar type names.",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/ObjectType"
+      }
+    },
+    "procedures": {
+      "description": "Procedures which are available for execution as part of mutations",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/ProcedureInfo"
+      }
+    },
+    "scalar_types": {
+      "description": "A list of scalar types which will be used as the types of collection columns",
+      "type": "object",
+      "additionalProperties": {
+        "$ref": "#/definitions/ScalarType"
+      }
+    }
+  },
+  "definitions": {
+    "AggregateFunctionDefinition": {
+      "description": "The definition of an aggregation function on a scalar type",
+      "type": "object",
+      "required": [
+        "result_type"
+      ],
+      "properties": {
+        "result_type": {
+          "description": "The scalar or object type of the result of this function",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Type"
+            }
+          ]
+        }
+      }
+    },
+    "ArgumentInfo": {
+      "type": "object",
+      "required": [
+        "type"
+      ],
+      "properties": {
+        "description": {
+          "description": "Argument description",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "type": {
+          "description": "The name of the type of this argument",
+          "allOf": [
+            {
+              "$ref": "#/definitions/Type"
+            }
+          ]
+        }
+      }
+    },
+    "CollectionInfo": {
+      "type": "object",
+      "required": [
+        "arguments",
+        "deletable",
+        "foreign_keys",
+        "name",
+        "type",
+        "uniqueness_constraints"
+      ],
+      "properties": {
+        "arguments": {
+          "description": "Any arguments that this collection requires",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/ArgumentInfo"
+          }
+        },
+        "deletable": {
+          "description": "Whether or not existing rows can be deleted from the collection",
+          "type": "boolean"
+        },
+        "description": {
+          "description": "Description of the collection",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "foreign_keys": {
+          "description": "Any foreign key constraints enforced on this collection",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/ForeignKeyConstraint"
+          }
+        },
+        "insertable_columns": {
+          "description": "The set of names of insercollection columns, or null if inserts are not supported",
```

Comment:
I think your find/replace went haywire here.

Reply:
Argh I tried so hard to avoid this exact replacement 😠